### PR TITLE
Update main README

### PR DESCRIPTION
**Pull request by hpmellema**
### Description of changes
Updates the main README to reflect changes to the plugin repository.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,192 @@
-# Smithy Gradle Plugin
+# Smithy Gradle Plugins
 [![Build Status](https://github.com/awslabs/smithy-gradle-plugin/workflows/ci/badge.svg)](https://github.com/awslabs/smithy-gradle-plugin/actions/workflows/ci.yml)
 
-This project integrates Smithy with Gradle. This plugin can build artifacts
-from Smithy models, generate JARs that contain Smithy models found in Java
-projects, and generate JARs that contain filtered *projections* of Smithy
-models.
+This project provides plugins to integrate [Smithy](https://smithy.io/2.0/index.html) with [Gradle](https://gradle.org/).
```
Comment (sugmanue) [inline README.md:4]:
```suggestion
This project provides plugins to integrate [Smithy](https://smithy.io/) with [Gradle](https://gradle.org/). 
```

No need to be specific with the path.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,192 @@
...
+
+See the [examples](./examples/jar-plugin) directory for examples of using this plugin.
+
+
+## Configuration
+### Customizing source directories  
+Smithy gradle plugins assumes Smithy model files (`*.smithy`) are organized in a similar way as Java source files, in sourceSets.
+The `smithy-base` plugin adds a new sources block named `smithy` to every sourceSet. By default, this source block will include 
+Smithy models in `model/`,`src/$sourceSetName/smithy` and`src/$sourceSetName/resources/META-INF/smithy`. New source directories can 
+be added to a `smithy` sources block as follows:
```
Comment (sugmanue) [inline README.md:87]:
```suggestion
be added to a `smithy` sources block as follows:

```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,192 @@
...
+
+
+### Set `smithy-build` configs to use
+By default, the plugin will look for a file called `smithy-build.json` at the 
+project's root and will use that as the [`smithy-build`](https://smithy.io/2.0/guides/building-models/build-config.html)
+config for your project. If no `smithy-build.json` file is found then a default 
+empty build config is used to build the project.
+
+Alternatively, you can explicitly configure one or more `smithy-build` configs to use 
+for your project as follows:
```
Comment (sugmanue) [inline README.md:146]:
```suggestion
for your project as follows:

```

File: README.md [hunk 4]
```diff
@@ -1,23 +1,192 @@
...
+// build.gradle.kts 
+smithy {
+    tags.addAll("tag1", "anotherTag", "anotherTag2")
+}
+```
+
+### Fork a new process when executing Smithy CLI commands
+By default, Smithy CLI commands are run in the same process as Gradle, but inside a thread with a custom class loader. 
+This should work in most cases, but there is an option to run inside a process if necessary. To run Smithy CLI commands
+in a process set the `fork` configuration option to `true`:
```
Comment (sugmanue) [inline README.md:170]:
```suggestion
in a process set the `fork` configuration option to `true`: 

```

File: README.md [hunk 5]
```diff
@@ -1,23 +1,192 @@
...
+    fork.set(true)
+}
+```
+
+
+### Disable Smithy Formatter
+By default, the `smithy format` CLI command is executed on all source directories. 
+This opinionated formatter follows the best practices recommended by the Smithy team. 
+It is possible to disable the formatter by setting the `format` setting on the plugin 
+extension to `false`:
```
Comment (sugmanue) [inline README.md:183]:
```suggestion
extension to `false`: 

```

**Review by sugmanue — APPROVED** [review]

**Review by AndrewFossAWS — APPROVED** [review]

**Pull request merged**